types/chrome-remote-interface/index.d.ts

```diff
@@ -45,6 +45,17 @@ declare namespace CDP {
         sessionId?: string | undefined;
     }
 
+    interface SendError {
+        code: number;
+        message: string;
+        data?: string | undefined;
+    }
+
+    interface SendCallback<T extends keyof ProtocolMappingApi.Commands> {
+        (error: true, response: SendError): void;
+        (error: false, response: ProtocolMappingApi.Commands[T]['returnType']): void;
```

Comment:
```suggestion
        (error: false, response: ProtocolMappingApi.Commands[T]['returnType'] | {}): void;
        (error: Error, response: undefined): void;
```

For the first line:
Code: https://github.com/cyrus-and/chrome-remote-interface/blob/v0.31.0/lib/chrome.js#L259
Documentation: none

For the second line:
Code: https://github.com/cyrus-and/chrome-remote-interface/blob/v0.31.0/lib/chrome.js#L290
Documentation: https://github.com/cyrus-and/chrome-remote-interface/tree/v0.31.0#clientsendmethod-params-sessionid-callback ... more specifically:

"In case of low-level WebSocket errors, the `error` parameter contains the originating `Error` object and no `response` is returned."

----
This should be the last review for this PR. Thanks for your patience and understanding.